internal/pkg/controllers/spod/setup.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spod
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/go-logr/logr"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"sigs.k8s.io/security-profiles-operator/internal/pkg/config"
+	"sigs.k8s.io/security-profiles-operator/internal/pkg/controllers/spod/bindata"
+)
+
+// TODO(jaosorior): Use better method than a label constant.
+const cmIsSPOdConfig = "security-profiles-operator/config"
+
+// SPOdTunables defines the parameters to tune/modify for the
+// Security-Profiles-Operator-Daemon.
+type DaemonTunables struct {
+	DaemonImage         string
+	NonRootEnablerImage string
+	WatchNamespace      string
+}
+
+// Setup adds a controller that reconciles seccomp profiles.
```

Comment:
reconciles seccomp profiles? Should this be reconciles the operator DaemonSet?

Reply:
fixed.